doc/manual.rst

```diff
@@ -7224,6 +7224,9 @@ The string literal passed to ``exportc`` can be a format string:
 In the example the external name of ``p`` is set to ``prefixp``. Only ``$1``
 is available and a literal dollar sign must be written as ``$$``.
 
+Should the symbol be exported to a dynamic library, the ``dynlib`` pragma should
```

Comment:
Hm. While this is grammatically correct, the phrasing is a bit... dated? A better sentence would be "If the symbol should also be exported to a dynamic library, the dynlib pragma should be used in addition to the exportc pragma".